recipes/rb-open_uri_redirections/LICENSE.txt

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2012-2014 Jaime Iniesta
```

Comment:
@svengiegerich, you should be able to get rid of the license file by following the same steps we discussed here: https://github.com/conda-forge/staged-recipes/pull/22205